### DISCUSSION TITLE
Tasks from querry visible in day planner

### DISCUSSION THREAD

**Discussion by Zavrod** [ivan-lednev/obsidian-day-planner#502] [Q&A]
Hi Ivan,
is there any chance I can get tasks from querries shown in the day planner side bar? I'd like to pull the tasks for today from whole vault and have at least the option to see it as a list on top of calendar. Do you know of any way how to do it?
Thanks
Jan

**Comment by dorahee** [comment]
I want to do that too! Couldn't find out how to do it!
